examples/async.rs

```diff
@@ -42,7 +42,13 @@ fn main() -> Result<(), Box<dyn std::error::Error>> {
     let backend = CrosstermBackend::new(stdout);
     let mut terminal = Terminal::new(backend)?;
 
+    #[cfg(all(feature = "rustix", unix))]
     let mut picker = Picker::from_termios()?;
+    #[cfg(not(all(feature = "rustix", unix)))]
+    let mut picker = {
+        let font_size = (8, 16);
+        Picker::new(font_size)
+    };
```

Comment:
If this is used to calculate the amount of space an image will occupy in character cells, it should be 10x20 for Windows Terminal (that's the cell size of the original VT340 hardware terminal).

You can also query the cell size with the `CSI 16 t` escape sequence.